bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/helpers/CommandHelpers.java

```diff
@@ -59,14 +58,14 @@ public static String getBookieSocketAddrStringRepresentation(BookieId bookidId,
            }
            realHostname = hostname;
         }
-        return formatBookieSocketAddress(bookieID, ip, networkAddress.getPort(), realHostname);
+        return formatBookieSocketAddress(bookidId, ip, networkAddress.getPort(), realHostname);
```

Comment:
nit: typo `bookidId` should be `bookieId`

Reply:
fixed. @Ghatage PTAL again